stubs/sortedcontainers/sortedset.pyi

```diff
@@ -0,0 +1,118 @@
+# stub for SortedDict. This is a lightly edited copy of
```

Comment:
```suggestion
# stub for SortedSet. This is a lightly edited copy of
```

What was edited? 😄 I didn't look super closely at these, but they look reasonable overall.

Reply:
I just isort/blacked it and copied that stub from the other files :)